apps/bsd/src/screens/BallotEjectScreen.tsx

```diff
@@ -136,12 +147,14 @@ const BallotEjectScreen: React.FC<Props> = ({
                   ? 'Overvote'
                   : isBlankSheet
                   ? 'Blank Ballot'
+                  : isInvalidElectionHashSheet
+                  ? 'Invalid Election'
```

Comment:
I know this is going along with the existing language, but maybe "Wrong Election" would be better?